include/glow/Backends/LayoutConverter.h

```diff
@@ -23,7 +23,7 @@ namespace glow {
 
 /// Convert regular convolution nodes (that use NHWC) into a backend-specific
 /// convolution nodes using NCHW.
-Node *convertConvToNCHWConv(ConvolutionNode *CN, Function *F) {
+inline Node *convertConvToNCHWConv(ConvolutionNode *CN, Function *F) {
```

Comment:
unrelated to the PR? (did it break ODR somewhere?)

Reply:
Yes, if two different backends include `LayoutConverter.h`, it breaks ODR and we get a linker error. Interestingly enough, this happens only with LLVM 9. When I compiled with LLVM 8, there were no such issues.